src/sentry/search/snuba/backend.py

```diff
@@ -493,6 +493,17 @@ def query(
         else:
             retention_window_start = None
 
+        logger.info(
+            "SnubaSearchBackendBase.query.start",
+            extra={
+                "organization_id": projects[0].organization_id,
```

Comment:
is this guaranteed to have at least one project?

Reply:
@vartec Yes, it would fail earlier in the function if there isn't exactly 1 org associated with every project